src/AI/Muriel.java

```diff
@@ -551,7 +551,7 @@ else if( obst.isTurnOver || unitSelector.isObstacle(obst) )
     //////////////////////////////////////////////////////////////////
     // We didn't find an immediate ATTACK or CAPTURE action we can do.
     // Things that can capture; go find something to capture, if you are moderately healthy.
-    if( unit.model.hasActionType(UnitActionFactory.CAPTURE) && (unit.getHP() >= 7) )
+    if( unit.hasActionType(UnitActionFactory.CAPTURE) && (unit.getHP() >= 7) )
```

Comment:
Do you not `calculatePossibleActions` here because we are just checking for a default action type?

Reply:
I do actually `calculatePossibleActions` - the new `Unit` copy of `hasActionType` invokes it.